samples/openapi3/client/petstore/python-experimental/docs/AnimalFarm.md

```diff
@@ -0,0 +1,9 @@
+# animal_farm.AnimalFarm
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
```

Comment:
Can you fix the doc template so it lists the information for the AnimalFarm model here?
Name: N/A
Type: [animal.Animal]

Reply:
The problem is that `{{#vars}}` is empty for arrays.

Reply:
@spacether solved - see https://github.com/OpenAPITools/openapi-generator/pull/6493/files#diff-ec679c7ef213544fd9e5fdad21ffd6eeR6